base/compiler/typelattice.jl

```diff
@@ -368,7 +404,8 @@ ignorelimited(typ::LimitedAccuracy) = typ.typ
 function ⊑(lattice::InferenceLattice, @nospecialize(a), @nospecialize(b))
     if isa(b, LimitedAccuracy)
         if !isa(a, LimitedAccuracy)
-            return false
+            # epsilon smaller
+            return ⊑(widenlattice(lattice), a, b.typ) && !⊑(widenlattice(lattice), b.typ, a)
         end
         if b.causes ⊈ a.causes
             return false
```

Comment:
This should reflect the epsilon case too, though I think that means refactoring part of this code for better ordering than this
```suggestion
            if ⊑(widenlattice(lattice), a.typ, b.typ)
                return true
            end
            return false
```

Reply:
(This also must be reflected now in the type-complexity tests. Where this is meaning the comparison of the length of causes.)

Reply:
> the type-complexity tests

You mean `issimplertype`? In the current design, that's not reached for LimitedAccuracy and before we were just passing those arguments through `ignorelimited`.

Reply:
Yeah. It seems a little odd for tmerge_fast_path not to be legal anymore for this type, but I guess that looks right. We should perhaps assert there since we know LimitedAccuracy may be handled wrong there now?

Reply:
We were correct before, since it was already implied by the precondition of suba or subb. That assumed implication is no longer valid after this change.

Reply:
Let me remove the `ignorelimited` in issimplertype then. We'll get an appropriate assertion if somebody accidentally uses it.